sklearn/metrics/cluster/tests/test_common.py

```diff
@@ -0,0 +1,219 @@
+# properties, e.g. invariance toward several input layout.
+#
+
+#metrics used to test similarity between bicluster
+BICLUSTER_METRICS = {
+    "consensus_score":consensus_score    
+    }
+      
+SUPERVISED_METRICS = {
+    "adjusted_mutual_info_score":adjusted_mutual_info_score ,
+    "adjusted_rand_score":adjusted_rand_score,
+    "completeness_score":completeness_score,
+    "contingency_matrix":contingency_matrix,
+    "homogeneity_score":homogeneity_score,
+    "expected_mutual_information":expected_mutual_information,
+    "homogeneity_completeness_v_measure":homogeneity_completeness_v_measure,
+    "mutual_info_score":mutual_info_score,
+    "normalized_mutual_info_score":normalized_mutual_info_score,
+    "v_measure_score":v_measure_score
+    }
+        
+UNSUPERVISED_METRICS = {
+    "silhouette_score":silhouette_score
+    }
+        
+ALL_METRICS = dict()
+ALL_METRICS.update(BICLUSTER_METRICS)
+ALL_METRICS.update(SUPERVISED_METRICS)
+ALL_METRICS.update(UNSUPERVISED_METRICS)
+
+# Lists of metrics with common properties
+# ---------------------------------------
+# Lists of metrics with common properties are used to test systematically some
+# functionalities and invariance, e.g. SYMMETRIC_METRICS lists all metrics that
+# are symmetric with respect to their input argument y_true and y_pred.
+#
+#---------------------------------------------------------------------
+# Symmetric with respect to their input arguments y_true and y_pred.
+# Symmetric metrics only imply to supervised clusters.
+SYMMETRIC_METRICS = [
+    "adjusted_rand_score","v_measure_score",
+    "mutual_info_score","adjusted_mutual_info_score",
+    "normalized_mutual_info_score",
+    ]
+                    
+NON_SYMMETRIC_METRICS = ["homogeneity_score" , "completeness_score"]
+
+# When the information is zero these metrics output zero.
+METRICS_ZERO_INFO = [
+    "normalized_mutual_info_score","v_measure_score",
+    "adjusted_mutual_info_score"
+    ]
+ 
+# Metrics with output between 0 and 1                                      
+METRICS_NORMALIZED_OUTPUT = [
+    "adjusted_rand_score","homogeneity_score","completeness_score",
+    "v_measure_score","adjusted_mutual_info_score",
+    "normalized_mutual_info_score"
+    ]                  
+                      
+# Metrics where permutations of labels dont change score( 0 and 1 exchchanged)
+METRICS_PERMUTE_LABELS = ["homogeneity_score","v_measure_score",
+    "completeness_score","mutual_info_score","adjusted_mutual_info_score",
+    "normalized_mutual_info_score","adjusted_rand_score"
+    ]
+
+# Input parameters can be both in the form of arrays and lists
+METRICS_WITH_FORMAT_INVARIANCE = ["homogeneity_score","v_measure_score",
+    "completeness_score","mutual_info_score","adjusted_mutual_info_score",
+    "normalized_mutual_info_score","adjusted_rand_score"
+    ] 
+    
+# If classes members are completely split across different clusters,

```

Comment:
I still don't get what the point of this group is. `adjusted_rand_accuracy` and `mutual_info_score` and `fowlkes_mallows_score` would also pass the test as it stands.